static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-assets-folder.js

```diff
@@ -1965,6 +1984,32 @@ var storage = CStudioAuthoring.Storage;
       );
     },
 
+    createClip: function () {
+      const wrapper = document.createElement('div');
+      const onClose = () => {
+        CrafterCMSNext.ReactDOM.unmountComponentAtNode(wrapper);
+      };
+
+      const src = oCurrentTextNode.data.browserUri;
+
+      CrafterCMSNext.render(wrapper, 'VideoPlayerDialog', {
+        id: 'dialogPlayer',
+        onClose: onClose,
```

Comment:
Need to unmount onClosed so that the animation is not cut.
The render function returns the unmount function already. Just store return value and use in onClosed prop.